keps/sig-storage/1855-csi-driver-service-account-token/kep.yaml

```diff
@@ -15,9 +15,9 @@ approvers:
   - "@msau42"
   - "@mikedanese"
 creation-date: 2020-06-09
-last-updated: 2020-07-07
+last-updated: 2020-11-12
 status: implementable
-stage: alpha
+stage: beta
```

Comment:
It's still alpha. I think when we plan to move to beta in 1.21, we can update this again along with the other beta kep requirements.

Reply:
done. thought it is implementable for beta. i changed to implemented for alpha.